### PR TITLE
Flap Check Logic Update - Airbus & Boeing 737-300/900 Series

### PR DESCRIPTION
**Pull request by FatihKoz**
Only update flap check for all airbus profiles, nothing more.

### CODE REVIEW

**Comment by Clamb94** [comment]
Should we also add tolerance for value 1.0 ? Like `> 0.95` instead of `== 1`

**Comment by FatihKoz** [comment]
Honestly, I am not using that kind of controller and can not test that possibility. Logically moving a flap lever/handle to its max setting (for flap full) will probably return 1.

Your safety solution is also fine, at least it will eliminate issues regarding full flap value mismatch. If you wish I can add that changes, also as the repository owner/maintainer you can edit my files too (before merging) as you wish.

**Review by Clamb94 — APPROVED** [review]

**Pull request merged**
